docs/source/algorithms/ReflectometryReductionOneAuto-v2.rst

```diff
@@ -0,0 +1,209 @@
+:ref:`algm-PolynomialCorrection` algorithm, with the polynomial string taken
+from the instrument's :literal:`polystring` parameter. If the
+:literal:`correction` parameter is set to :literal:`exponential` instead, then
+the :Ref:`algm-ExponentialCorrection` algorithm is used, with C0 and C1 taken
+from the instrument parameters, :literal:`C0` and :literal:`C1`. All these values
+can be specified manually by setting the :literal:`CorrectionAlgorithm` to either
+:literal:`PolynomialCorrection` or :literal:`ExponentialCorrection` and setting
+:literal:`Polynomial` or :literal:`C0` and :literal:`C1` properties accordingly.
+Note that when using a correction algorithm, monitors will not be integrated, even if
+:literal:`NormalizeByIntegratedMonitors` was set to true.
+
+Finally, properties :literal:`MomentumTransferMin`, :literal:`MomentumTransferStep` and :literal:`MomentumTransferMax` are
+used to rebin the output workspace in Q, and :literal:`ScaleFactor` is used to scale the rebinned workspace. When they
+are not provided the algorithm will attempt to determine the bin width using :ref:`algm-CalculateResolution` (note that, for
+the latter to run successfully, a :literal:`slit` component with a :literal:`vertical gap` must be defined in the
+instrument definition file).
+
+See :ref:`algm-ReflectometryReductionOne` for more information
+on how the input properties are used by the wrapped algorithm.
+
+Workspace Groups
+################
+
+If a workspace group is provided as input, each workspace in the group will be
+reduced independently and sequentially using :ref:`algm-ReflectometryReductionOne`. Each of these
+individual reductions will produce three output workspaces: an output workspace in
+wavelength, an output workspace in Q with native binning, and a rebinned workspace in Q.
+Output workspaces in wavelength will be grouped together to produce an output workspace group in wavelength, and output
+workspaces in Q will be grouped together to produce an output workspace group in Q.
+The diagram below illustrates this process (note that, for the sake of clarity, the rebinned output
+workspace in Q, :literal:`OutputWorkspaceBinned`, is not represented but it is handled analogously to
+:literal:`OutputWorkspace` and :literal:`OutputWorkspaceWavelength`):
+
+.. diagram:: ReflectometryReductionOneAuto-v2-Groups_noPA_wkflw.dot
+
+Polarization Analysis Off
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If :literal:`PolarizationAnalysis = None` the reduction stops here. Note that if
+transmission runs are given in the form of a workspace group, each member in the
+transmission group will be associated to the corresponding member in the input
+workspace group, i.e., the first item in the transmission group will be used as the
+transmission run for the first workspace in the input workspace group, the second
+element in the transmission group will be used as the transmission run for the
+second workspace in the input workspace group, etc. This is also illustrated
```

Comment:
I feel that it should also be mentioned that if only one item is given for 1st/2nd transmission run, that run would be used for all members of the input workspace group.